lib/l10n/intl_en.arb

```diff
@@ -18,7 +18,7 @@
     "placeholders_order": [],
     "placeholders": {}
   },
-  "username": "username",
+  "username": "Email",
```

Comment:
why you not change the variable name `username` to `email` too?

Reply:
Thought about it, but wasn't sure about all the links in the code... But it would make sense I agree, will give it a shot